sklearn/utils/multiclass.py

```diff
@@ -247,7 +247,10 @@ def type_of_target(y):
         return 'multilabel-indicator'
 
     try:
-        y = np.asarray(y)
+        if not issparse(y):
+            y = np.asarray(y)
+        else:
```

Comment:
Well, I'd still like this to return unknown seeing as we don't support sparse arrays as a representation for multiclass multioutput... I was just trying to clarify your use of terminologies.